### PR TITLE
Bug/sc 32510/editing a learning object points to a previous

### PR DESCRIPTION
**Pull request by curlyLasagna**
Add version to the URL to point to the correct learning object through `cuid` and `version`, instead of `learningObjectId` which is no longer being sent to the URL

### CODE REVIEW

File: src/app/onion/learning-object-builder/components/content-upload/app/upload/upload.component.ts [hunk 1]
```diff
@@ -654,6 +654,7 @@ export class UploadComponent implements OnInit, AfterViewInit, OnDestroy {
    * @memberof UploadComponent
    */
   private handleUploadError(update: UploadErrorUpdate) {
+    console.log(update);
```
Comment (Greek) [inline src/app/onion/learning-object-builder/components/content-upload/app/upload/upload.component.ts:657]:
```suggestion

```

Comment (Greek) [inline src/app/onion/learning-object-builder/components/content-upload/app/upload/upload.component.ts:657]:
```suggestion

```

**Review by Greek — APPROVED** [review]
just a console.log

File: src/app/core/learning-object-module/uri-retriever.service.ts [hunk 2]
```diff
@@ -207,8 +207,6 @@ export class UriRetrieverService {
     let route;
     // Sets route to be hit based on if the id or if Learning Object name have been provided
     if (params.cuidInfo?.cuid) {
-      route = LEGACY_USER_ROUTES.GET_LEARNING_OBJECT(params.cuidInfo.cuid);
```
Comment (pzalep1) [inline src/app/core/learning-object-module/uri-retriever.service.ts:210]: image

**Review by pzalep1 — APPROVED** [review]

**Review by stef359 — APPROVED** [review]

**Review by johngonz0 — APPROVED** [review]

**Pull request merged**